### ISSUE TITLE
FATAL EXCEPTION: main trying to remove microphone gain

### ISSUE THREAD

**Issue by 532910** [BelledonneCommunications/linphone-android#932]
![ScreenRecord-2020-03-03-01-05-40](https://user-images.githubusercontent.com/14617699/75724502-94b43a80-5cef-11ea-8113-ea78e048d7a1.gif)

```
E AndroidRuntime: FATAL EXCEPTION: main
E AndroidRuntime: Process: org.linphone, PID: 7424
E AndroidRuntime: java.lang.NumberFormatException: empty String
E AndroidRuntime: 	at sun.misc.FloatingDecimal.readJavaFormatString(FloatingDecimal.java:1842)
E AndroidRuntime: 	at sun.misc.FloatingDecimal.parseFloat(FloatingDecimal.java:122)
E AndroidRuntime: 	at java.lang.Float.parseFloat(Float.java:451)
E AndroidRuntime: 	at java.lang.Float.valueOf(Float.java:416)
E AndroidRuntime: 	at org.linphone.settings.c$c.a(AudioSettingsFragment.java:1)
E AndroidRuntime: 	at org.linphone.settings.widget.TextSetting$a.onFocusChange(TextSetting.java:2)
E AndroidRuntime: 	at android.view.View.onFocusChanged(View.java:7265)
E AndroidRuntime: 	at android.widget.TextView.onFocusChanged(TextView.java:10022)
E AndroidRuntime: 	at android.view.View.clearFocusInternal(View.java:7101)
E AndroidRuntime: 	at android.view.View.unFocus(View.java:7134)
E AndroidRuntime: 	at android.view.ViewGroup.unFocus(ViewGroup.java:1155)
E AndroidRuntime: 	at android.view.ViewGroup.unFocus(ViewGroup.java:1155)
E AndroidRuntime: 	at android.view.ViewGroup.unFocus(ViewGroup.java:1155)
E AndroidRuntime: 	at android.view.ViewGroup.unFocus(ViewGroup.java:1155)
E AndroidRuntime: 	at android.view.ViewGroup.unFocus(ViewGroup.java:1155)
E AndroidRuntime: 	at android.view.ViewGroup.removeViewInternal(ViewGroup.java:5343)
E AndroidRuntime: 	at android.view.ViewGroup.removeViewInternal(ViewGroup.java:5330)
E AndroidRuntime: 	at android.view.ViewGroup.removeView(ViewGroup.java:5261)
E AndroidRuntime: 	at android.app.FragmentManagerImpl.moveToState(FragmentManager.java:1419)
E AndroidRuntime: 	at android.app.FragmentManagerImpl.moveFragmentToExpectedState(FragmentManager.java:1576)
E AndroidRuntime: 	at android.app.BackStackRecord.executePopOps(BackStackRecord.java:861)
E AndroidRuntime: 	at android.app.FragmentManagerImpl.executeOps(FragmentManager.java:2402)
E AndroidRuntime: 	at android.app.FragmentManagerImpl.executeOpsTogether(FragmentManager.java:2200)
E AndroidRuntime: 	at android.app.FragmentManagerImpl.removeRedundantOperationsAndExecute(FragmentManager.java:2155)
E AndroidRuntime: 	at android.app.FragmentManagerImpl.popBackStackImmediate(FragmentManager.java:887)
E AndroidRuntime: 	at android.app.FragmentManagerImpl.popBackStackImmediate(FragmentManager.java:831)
E AndroidRuntime: 	at org.linphone.activities.b.E(MainActivity.java:2)
E AndroidRuntime: 	at org.linphone.settings.SettingsActivity.y(SettingsActivity.java:2)
E AndroidRuntime: 	at org.linphone.activities.b$g.onClick(MainActivity.java:1)
E AndroidRuntime: 	at android.view.View.performClick(View.java:6597)
E AndroidRuntime: 	at android.view.View.performClickInternal(View.java:6574)
E AndroidRuntime: 	at android.view.View.access$3100(View.java:778)
E AndroidRuntime: 	at android.view.View$PerformClick.run(View.java:25906)
E AndroidRuntime: 	at android.os.Handler.handleCallback(Handler.java:873)
E AndroidRuntime: 	at android.os.Handler.dispatchMessage(Handler.java:99)
E AndroidRuntime: 	at android.os.Looper.loop(Looper.java:193)
E AndroidRuntime: 	at android.app.ActivityThread.main(ActivityThread.java:6718)
E AndroidRuntime: 	at java.lang.reflect.Method.invoke(Native Method)
E AndroidRuntime: 	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:491)
E AndroidRuntime: 	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:858)
```

**Comment by Viish** [comment]
Already fixed in a42d7a47aebdd08a9a0f23b2b9bba6e476236425

**Comment by 532910** [comment]
ack
